### ISSUE TITLE
Question regarding license

### ISSUE THREAD

**Issue by salazarbarrera** [jakeledoux/live#3]
tl;dr: Can I do derivative work and distribute it? Can you explicitly add a license file to the code?

---
Hi! I downloaded and slightly modified your [website](https://jakeledoux.com/live/) so I could upload a fully client-side instance of it on neocities (with no php). This because I noticed that sometimes this message could be read in the console:

> "Rate Limit Exceeded - This application has made too many requests in a short period. If this is your API key, see https://www.last.fm/api/tos#4.4 for information about raising the limit

So, in order to free some load from your own key, I decided to "run" my own instance with my own API key. [As you can see](https://salazarbarrera.neocities.org/ListenAlongJS/live/), I actually did it, but only now I realize your program is not AGPL as I initially assumed (I noticed when I wanted to upload the license to my site).

Am I complying with the intentions you had when you published this source code? Or do you prefer I take down this instance and refrain from distributing my modified code?

**Comment by jakeledoux** [comment]
Sorry for not responding sooner - I’m not on GitHub as much as I used to be. What you’re doing is totally fine by me! I need to rewrite this whole thing from scratch sometime soon, the code is so gross and PHP really isn’t necessary. So I’ll probably be throwing it all away anyway. Thanks for asking!
